docs/conf.py

```diff
@@ -51,11 +51,11 @@ def nothing(*arg):
 extensions = [
     'sphinx.ext.autodoc',
     'sphinx.ext.doctest',
+    'repoze.sphinx.autointerface',
```

Comment:
Is there a particular reasoning for the ordering of these? Or can they be alphabetized?